lock/src/test/java/org/infinispan/lock/ClusteredLockWith2NodesKillingOneTest.java

```diff
@@ -0,0 +1,59 @@
+package org.infinispan.lock;
+
+import static org.infinispan.functional.FunctionalTestUtils.await;
+import static org.infinispan.test.TestingUtil.killCacheManagers;
+import static org.testng.Assert.assertEquals;
```

Comment:
We tend to favor the AssertJUnit versions of these

Reply:
Wrong import by mistake. Let me change and push